src/App.scss

```diff
@@ -53,6 +53,11 @@
   }
 }
 
+:global(.list-full-height) {
+  height: 100vh !important;
+  min-height: 100vh !important;
+}
+
```

Comment:
Better to minimise use of global classes. This class can be a part of css module `src/ducks/Watchlists/Widgets/Table/index.scss` that will be added directly to the app's root DOM element (more info here `[D-0-1]`)

Reply:
@DmitriVanGuard 
Viewport height is set in `src/App.js` file by:

```
    <div
      className={cx(
        'App',
        isWatchlistPage && isDesktop && 'list-container',
        isWatchlistPage && isDesktop && isLessThan20 && 'list-full-height'
      )}
    > 
```

How is it possible to change App class in the `src/ducks/Watchlists/Widgets/Table` file?